.cm/gitstream.cm

```diff
@@ -0,0 +1,77 @@
+# -*- mode: yaml -*-
+# This example configuration for provides basic automations to get started with gitStream.
+# View the gitStream quickstart for more examples: https://docs.gitstream.cm/examples/
+manifest:
+  version: 1.0
+
+
+triggers:
+  exclude:
+    branch:
+      - l10n_dev
+      - dev
+
+
+automations:
+  # Add a label that indicates how many minutes it will take to review the PR.
+  estimated_time_to_review:
+    if:
+      - true
+    run:
+      - action: add-label@v1
+        args:
+          label: "{{ calc.etr }} min review"
+          color: {{ colors.red if (calc.etr >= 20) else ( colors.yellow if (calc.etr >= 5) else colors.green ) }}
+  # Post a comment that lists the best experts for the files that were modified.
```

Comment:
@coderabbitai Provide the complete list of all merged PRs for milestone 1.19.0 with the enhancement or bug label, making sure to handle pagination or use a limit to capture all results.

Reply:
This is not the complete list

Reply:
Why ere you able to get the complete list this time?

Reply:
What do I say to get this result next time?